File: app/src/main/java/org/blitzortung/android/app/BootReceiver.java
package org.blitzortung.android.app;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Intent bootIntent = new Intent(context, AppService.class);
        bootIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        context.startService(bootIntent);
    }
}

File: app/src/main/java/org/blitzortung/android/app/BootReceiver.kt
package org.blitzortung.android.app

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val bootIntent = Intent(context, AppService::class.java)
        bootIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        context.startService(bootIntent)
    }
}